### DISCUSSION TITLE
Add the ability to login with freecodecamp login

### DISCUSSION THREAD

**Discussion by kingpinzs** [freeCodeCamp/Developer_Quiz_Site#256] [Ideas]
Allow users to login if they want to so they can track progress stored in local cash for now.

**Comment by jdwilkin4** [comment]
I am not 100% on board with integrating user's freeCodeCamp login into the quiz site and changing where the site is deployed to support that new login. 

If the goal is to just keep previous high scores, we could just use `localStorage` and not bother with the login piece.

But I will also bring in other freeCodeCamp team members that were involved with the early development of the site if they are interested in sharing their thoughts. 👍 

@naomi-lgbt @deniselemonaki @estefaniacn

**Reply by naomi-lgbt** [reply]
Adding authentication would unlock some really neat potential, such as being able to display completion badges on a camper's `/learn` profile.

But setting that up would be a heavy task, and I'm not sure it's a high enough priority for us to invest our limited developer bandwidth in at this time.

**Comment by kingpinzs** [comment]
I think storing it locally would be a great start.

**Comment by jdwilkin4** [comment]
Let's keep it simple and just have a list of the user's high scores on the results page and store it in `localStorage`.
You are free to create an issue with details for this feature 👍

**Reply by Verifiedthug** [reply]
Hello I'm new here can you enlighten me more no this

**Comment by vvs-basanth** [comment]
Hello @jdwilkin4,

I wanted to check if the local storage feature has already been implemented in this project. I reviewed a few quiz questions but couldn't find any related functionality on the website. It's possible I may have overlooked something.

If this feature hasn’t been worked on yet, I'd be glad to take on the task. 

Please let me know the current status.

Thank you.

**Reply by jdwilkin4** [reply]
No this feature has not been added.

If you are interested, you will need to start by creating an issue detailing what this feature will entail concerning the localStorage aspect.
Then, one of the maintainers will approve the requested feature and we can assign it to you

**Reply by vvs-basanth** [reply]
Will do that right away!

Thank You!

**Reply by vvs-basanth** [reply]
Hey,
I created an issue detailing what exactly I will be working on [here](https://github.com/freeCodeCamp/Developer_Quiz_Site/issues/1055).

Thanx
